manifest.json

```diff
@@ -3,49 +3,47 @@
     "id": "ihatemoney",
     "packaging_format": 1,
     "description": {
-        "en": "A simple shared budget manager web application",
-        "fr": "Une application web de comptes partagés à plusieurs"
+        "en": "Simple app to manage your collective expenses",
+        "fr": "Gérez simplement vos dépenses collectives"
     },
     "url": "http://ihatemoney.org/",
-    "version": "4.1.5~ynh2",
-    "license": "free",
+    "version": "4.1.5~ynh3",
+    "license": "Beerware",
+    "upstream": {
+        "license": "MIT",
```

Comment:
Eeeh these two license fields are supposed to be the same

Reply:
zblerg nope, both are the upstream license ... the fact that we have an "upstream" dict is still pretty new and we kept the old license key at the root of the manifest mainly for backward compat ... Will be much cleaner in manifest v2..

Reply:
OK !

Reply:
I don't have the permissions on the testing branch :(